README.md

```diff
@@ -1,158 +1,236 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+description: "This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure."
+urlFragment: key-vault-node-quickstart
+---
 
 # Quickstart: Set and retrieve a secret from Azure Key Vault using a Node Web App 
 
-This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure. The quickstart uses Node.js and Managed service identities (MSIs)
+This Quickstart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure. The Quickstart uses Node.js and Azure Managed Identities
 
 > * Create a Key Vault.
 > * Store a secret in Key Vault.
 > * Retrieve a secret from Key Vault.
 > * Create an Azure Web Application.
-> * [Enable managed service identities](https://docs.microsoft.com/azure/active-directory/managed-service-identity/overview).
+> * [Enable Azure Managed Identities](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/).
 > * Grant the required permissions for the web application to read data from Key vault.
 
-Before you proceed make sure that you are familiar with the [basic concepts](key-vault-whatis.md#basic-concepts).
+Before you proceed make sure that you are familiar with the [basic concepts](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-overview).
+
+# SDK Versions
+
+You will find the following folders: key-vault-node-quickstart-v3, which references the version 3.0 SDK and keyvault-node-quickstart-v4, which uses the version 4.0 SDK.
```

Comment:
In this sample, you will find the following folders: 

 - **key-vault-node-quickstart-v3**  - references Key Vault SDK v3.0 
 - **keyvault-node-quickstart-v4** - references Key Vault SDK v4.0